### ISSUE TITLE
Add option `between-group` to `scss/dollar-variable-empty-line-before`

### ISSUE THREAD

**Issue by evilebottnawi** [kristerkari/stylelint-scss#94]
Maybe best name. I love this rule, But sometimes i am use newlines between groups example:
```scss
$float-label-transition-duration: 0.25s !default;
$float-label-background: $input-bg !default;
$float-label-color: $input-color-placeholder !default;

$float-label-filled-color: $text-color !default;
$float-label-filled-height: $font-size-base !default;
$float-label-filled-padding: 0 !default;
$float-label-filled-font-size: 75% !default;

$float-label-focus-color: $brand-primary;
```
For a better reading, it would be convenient to have an option that disables checking between variables.

**Comment by dryoma** [comment]
Hmm, that's acrually reasonable, but could be quite subjective. Should it only check dash-delimited letter groups? What if someone prefers camelCase/PascalCase for their variables?

**Comment by evilebottnawi** [comment]
@dryoma i think we can check only if before i am using variable
```scss
$float-label-color: $input-color-placeholder !default; // here variable

$float-label-filled-color: $text-color !default; // we don't throw error on this line if i set `between-group`
```
It does not matter how the variable is written - camelCase, PascalCase :smile: But maybe add `after-dollar-variable` in `ignore` is also a solution, the most important thing that I want a blank line before the variables, and between variables (groups as in example).
